runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -98,6 +98,91 @@ function M.on_refresh(err, _, ctx, _)
   return vim.NIL
 end
 
+--- @class InlayHintOptions
+--- @field bufnr number|nil
+--- @field filter function|nil
+--- @field id number|nil
+--- @field name string|nil
+--- @field range { start: number[], end: number[] }|nil
+
+--- Returns inlay hints in the (optionally) given buffer, client, and range.
+--- This method does not make an LSP request, but just queries locally stored inlay hints.
+---
+--- @param options table|nil Optional table which holds the following optional fields:
```

Comment:
Nit: It seems like these are usually just named `opts`.

Reply:
Updated, thanks!